README.md

```diff
@@ -11,58 +11,58 @@ This experimental application is published for examination and evaluation. It is
 
 ### Logon to container
 
-    docker-compose run --rm --no-deps --entrypoint /bin/bash node
```

Comment:
`--no-deps` is superfluous because there’s only one container defined in `docker-compose.yml`.